be/src/olap/tablet.cpp

```diff
@@ -577,9 +578,9 @@ OLAPStatus Tablet::capture_consistent_versions(const Version& spec_version,
     return status;
 }
 
-OLAPStatus Tablet::check_version_integrity(const Version& version) {
+OLAPStatus Tablet::check_version_integrity(const Version& version, bool quite) {
```

Comment:
what does quite mean?

Reply:
if !quite, it will print log for missing versions.

Reply:
You mean quiet? slient may be better